### PR TITLE
clusterinfo support: Try to register prometheus.

### PR DESCRIPTION
**Pull request by mapleFU**
Signed-off-by: mapleFU <1506118561@qq.com>

### What problem does this PR solve? 

Register Prometheus message to pd's etcd. Fix this issue: https://github.com/pingcap-incubator/tidb-dashboard/issues/222

### What is changed and how it works?

Add codes in playground.

### Check List 

Tests 

 - No code

Code changes

 - Has interface methods change

Side effects

 - Possible performance regression
 - Increased code complexity

### CODE REVIEW

File: components/playground/main.go [hunk 1]
```diff
@@ -17,6 +17,11 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"go.uber.org/zap"
```
Comment (lonng) [inline components/playground/main.go]: Please optimize the imports.

File: components/playground/main.go [hunk 2]
```diff
@@ -287,6 +289,12 @@ func bootCluster(version string, pdNum, tidbNum, tikvNum int, host string, monit
 			return err
 		}
 	}
```
Comment (breeswish) [inline components/playground/main.go]: Line 275 ~ Line 283 can be removed?

Comment (mapleFU) [reply components/playground/main.go]: Yes, I'll remove them.

Comment (mapleFU) [reply components/playground/main.go]: Updated, PTAL.

**Pull request merged**